package.json

```diff
@@ -20,7 +20,8 @@
   },
   "homepage": "https://github.com/IgnacioBarocchi/HumbleScraper#readme",
   "dependencies": {
-    "node-fetch": "^2.6.0"
+    "node-fetch": "^2.6.0",
+    "yargs": "^15.3.1"
```

Comment:
You didn't remove `yargs`, please double check all comments before pushing new updates. Let me know if you're having trouble finding what the pending comments are and we can look at it together.